homeassistant/components/ipma/weather.py

```diff
@@ -20,7 +20,7 @@
 from homeassistant.helpers import config_validation as cv
 from homeassistant.util import Throttle
 
-REQUIREMENTS = ['pyipma==1.1.6']
+REQUIREMENTS = ['pyipma==1.2.1']
```

Comment:
Does this need to move to the component for the config entry to work?

Reply:
No (I've tested)

There is no validation from pyipma of the information provided in the config flow, so no need to have a requirement there.